src/Bicep.Core/PrettyPrintV2/SyntaxLayouts.cs

```diff
@@ -637,18 +647,63 @@ private IEnumerable<Document> LayoutParenthesizedTypeSyntax(ParenthesizedTypeSyn
             this.LayoutMany(leadingNodes)
                 .Where(x => x != HardLine); // Remove empty lines between decorators.
 
-        private IEnumerable<Document> LayoutCommaSeparatedArguments<T>(ImmutableArray<SyntaxBase> children, ImmutableArray<T> arguments)
+        private Document? LayoutFunctionTailObjectOrArrayArgumentSyntax(ImmutableArray<FunctionArgumentSyntax> argumentSyntaxes)
```

Comment:
nit: "layout" is a noun, "lay out" is a verb.  But it is a free country, so you decide.... :-)

Reply:
Yeah good catch. Will probably change all occurrence of `Layout...` to `LayOut...` in a separate PR.